gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -64,9 +64,8 @@ def test_image_pipe(tmpdir):
     mosaic = MosaicImage(image, energy_band=energy_band, offset_band=offset_band, data_store=data_store,
                          obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
     mosaic.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
-    # TODO: this fails on some machines, for unknown reasons.
-    #assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
-    #assert_almost_equal(mosaic.maps['bkg'].data.sum(), 1987.2088283446637)
-    #assert_almost_equal(mosaic.maps['exposure'].data.sum(), 5.9292301086257267e17)
-    #assert_almost_equal(mosaic.maps['significance'].lookup(center), 33.699068269123039)
-    #assert_almost_equal(mosaic.maps['excess'].data.sum(), 346.79117165533626)
+    assert_allclose(mosaic.maps['counts'].data.sum(), 2334.0, atol=3)
```

Comment:
I'm not sure if it's acceptable to have a tolerance of +/- 3 counts for this test. In the worst case this can make the difference between a detection or non-detection of a source. Actually the total number of events should be the same on all machines. Of course there are floating point differences, but if this leads to different results, when the event lists are binned into sky maps, this probably means the event directions are stored with too high precision. Right? Maybe rounding to a reasonable precision before binning events into a sky map could solve the problem?